README.md

```diff
@@ -14,21 +13,60 @@ Takeaway Challenge
 
  ```
 
```

Comment:
README is very clear with very detailed instructions in setting up and using the program